### ISSUE TITLE
Documentation + website: say that PHP 5.4 is needed

### ISSUE THREAD

**Issue by knbknb** [thephpleague/flysystem#645]
Neither the documentation on github nor the website http://flysystem.thephpleague.com/ state anywhere clearly that flysystem won't work with PHP 5.3. At least PHP 5.4 is needed. 

I for one didn't find this info.

Giving developers this info is important because there exist many older websites that still run PHP 5.3 or lower.  Developers hired for maintenance of these old sites, or on contract for pimping them need to know that flysystem won't run on webservers linked to PHP 5.3.

**Comment by frankdejonge** [comment]
@knbknb the PHP requirement is stated in the composer.json, it's also visible on the packagist page, which is liked to in the documentation and in the readme. Also, when you install it, composer will reject is because of the php version mismatch.

**Comment by frankdejonge** [comment]
I've added a min php version badge.

**Comment by knbknb** [comment]
Thanks!  Composer and packagist were not an option for me - I used the spl_autoload_register()-based mechanism as described on the http://flysystem.thephpleague.com/ homepage. Then I browsed mainly the documentation there.

  I didn't even try to use composer/packagist as I don't have shell access to the production server. 

Specifically I needed to add flysystem to a custom extension (=plugin) of the  TYPO3 CMS which is installed via uploading a zipped folder.
